cypress/integration/feedback.spec.js

```diff
@@ -0,0 +1,80 @@
+/* global describe, it, beforeEach, cy, expect */
+/* eslint-disable-next-line */
+import React from 'react'
+import fakeData from '../../js/data/report.json'
+import sampleRubric from '../../public/sample-rubric'
+
+describe('Provide Feedback', function () {
+  beforeEach(() => {
+    // cypress currently doesn't handle window.fetch so:
+    // diable browser's fetch this way the fetch polyfill is used which uses XHR
+    cy.on('window:before:load', win => {
+      win.fetch = null
+    })
+
+    cy.server()
+
+    // Return the fake report data
+    cy.route({
+      method: 'GET',
+      url: '/',
+      response: fakeData
+    })
+
+    // Fake report data references sample-rubric.json file, so it has to be stubbed too
+    cy.route({
+      method: 'GET',
+      url: '/sample-rubric.json',
+      response: sampleRubric
+    })
+
+    // On first load portal-report does a PUT request to save some report settings
+    cy.route({
+      method: 'PUT',
+      url: '/',
+      response: {}
+    }).as('putReportSettings')
+
+    let fakeServer = 'http://portal.test'
+    cy.visit(`http://localhost:8080/?reportUrl=${encodeURIComponent(fakeServer)}`)
```

Comment:
I think we shouldn't provide a hostname in the `visit` calls, so tests don't depend on dev setup and so it's easier to test against staging or production too. We can set `baseUrl` option in cypress.json file and then just call:
```cy.visit(`/?reportUrl=${encodeURIComponent(fakeServer)}`)```
Then, I can use CYPRESS_BASE_URL env variable to point it to my local server (I'm using 8081 port), staging or production. Or we can provide some more fancy configuration in the future (like here https://github.com/concord-consortium/lara-cypress-tests/pull/2).